CHANGELOG.md

```diff
@@ -4,6 +4,39 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.0] - 2022-10-18
+### Added
```

Comment:
We'll need to add `Breaking Changes`, since there were a lot of API changes that will break existing apps. It could be as simple as "The APIs are different, see commit/PR X for how to migrate".

Reply:
Added `Breaking Changes`, please revisit.

Reply:
Looks good, thanks